.github/workflows/ci.yml

```diff
@@ -196,15 +196,26 @@ jobs:
             backend: cpp
             env: { MACOSX_DEPLOYMENT_TARGET: 10.14 }
 
+        exclude:
+          # fails due to lack of a compatible compiler
+          - os: windows-2019
+            python-version: 2.7
+          - os: windows-2019
+            python-version: 3.4
+
+          # cpp specific test fails
+          - os: windows-2019
+            python-version: 3.5
+            backend: cpp
```

Comment:
Should we add `window_cpp_bugs_35.txt` for that?

```
======================================================================
FAIL: runTest (__mp_main__.EndToEndTest)
End-to-end cpdef_scoped_enums_import
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\cython\cython\runtests.py", line 1932, in runTest
    self.assertEqual(0, res, "non-zero exit status")
AssertionError: 0 != 1 : non-zero exit status
```

it doesn't seem to be very important tho

Reply:
I can't find a build where that shows up. Could you provide the failure details? It looks like something that might be a real bug.

Reply:
[Here](https://github.com/cython/cython/runs/3722392311?check_suite_focus=true), for example, but it seems like in the lstest tests that you've run its not present (at least no 3.5 job is failing)

Reply:
[There's](https://github.com/0dminnimda/cython/runs/4011113928?check_suite_focus=true) the latest ci run before your commit, also latest one with this error